src/routes/clinical-proxy.ts

```diff
@@ -119,4 +119,29 @@ router.use(
   }),
 );
 
+router.use(
+  '/program/:programId/clinical-data-tsv',
+  createProxyMiddleware({
+    target: CLINICAL_SERVICE_ROOT,
+    pathRewrite: (pathName: string, req: Request) => {
+      const programId = req.params.programId;
+      return urlJoin(
+        '/clinical/program/',
+        programId,
+        '/clinical-tsv',
+        `?donorIds=${req.query.donorIds}`,
+        req.query.submitterDonorIds
+          ? `&submitterDonorIds=${req.query.submitterDonorIds}`
+          : '',
+        req.query.entityTypes ? `&entityTypes=${req.query.entityTypes}` : '',
+        req.query.completionState
+          ? `&completionState=${req.query.completionState}`
+          : '',
```

Comment:
surely this can be simplified, can we try something like:
```js
const queryParams = Object.entries(req.query).map(([key, value]) => `${key}=${value}`).join('&');
return urlJoin(
    '/clinical/program/',
    programId,
    '/clinical-tsv',
    `?${queryParams}`,
```

Reply:
lol yeah this was top on my list of code clean up for today ;) Updated + working w/ your solution